lib/otw_sanitize/embed_sanitizer.rb

```diff
@@ -7,7 +7,6 @@ module OtwSanitize
   # Creates a Sanitize transformer to sanitize embedded media
   class EmbedSanitizer
     WHITELIST_REGEXES = {
-      ao3:              %r{^archiveofourown\.org/},
```

Comment:
`ao3` should also be removed from `ALLOWS_FLASHVARS` and `SUPPORTS_HTTPS`, since we're no longer allowing embeds from it at all.